chat-plugins/info.js

```diff
@@ -26,9 +26,6 @@ var commands = exports.commands = {
 			return this.errorReply("User " + this.targetUsername + " not found.");
 		}
 		var showAll = (cmd === 'ip' || cmd === 'whoare' || cmd === 'alt' || cmd === 'alts');
-		if (showAll && !user.can('lock') && targetUser !== user) {
```

Comment:
``user.autoconfirmed`` will return a users ``user.userid`` if they are confirmed, and ``undefined`` if they are not.